content/departments/tech-ops/tools/computer-setup.md

```diff
@@ -1,28 +1,27 @@
-## Computer requirements
+# Computer requirements
 
 - All full-time teammates must have a Sourcegraph computer
 - All teammates must work from their Sourcegraph computer
-- All teammates must use their Sourcegraph computer for accessing any company data
-- All teammates should adhear to our related usage and access policies:
-  - [Device useage and privacy policy](../process/team_device_usage_privacy.md)
+- All teammates must only use their Sourcegraph computer for accessing [company data](#company-data)
+- Sourcegraph approves the use of Apple MacOs only (you may work with Windows and Linux devices for work that does not involve company data, such as open source coding)
+- All teammates should adhere to our related usage and access policies:
+  - [Device usage and privacy policy](../process/team_device_usage_privacy.md)
   - [Access control policy](../../../company-info-and-process/policies/access-control-policy.md)
   - [Protect company assets and use them properly - Code of conduct](../../../company-info-and-process/communication/code_of_conduct.md#respect-others-and-their-property-and-confidential-information)
 
-# Company data
+## Company data
 
 Company data is any personal, confidential, or sensitive teammate or client information. This would include (but is not limited to): customer data and metadata, Sourcegraph IP, audit logs, security incidents, contracts, personal information about clients or teammates, and internal communications, such as email and Slack.
 
+We are currently working on defining our data classification policy which will further identify our data into groups and outline exceptions to this policy, such as the use of mobile phones/tablets. You may continue to access Gmail, calendar, and Slack on your mobile device as long as it is password protected, screen lock enabled, updated with the latest software, and is not jailbroken.
```

Comment:
Our data management policy already outlines classifications and has examples of data in it. I would maybe change it to something like " We are currently improving our Acceptable Use Policy, which will give further details on data handling expectations and device usage, such as use of mobile phones/tablets. .....